iceoryx_posh/test/moduletests/test_popo_toml_gateway_config_parser.cpp

```diff
@@ -27,6 +26,13 @@
 #include <cpptoml.h>
 #include <limits> // workaround for missing include in cpptoml.h
 
+#if __cplusplus >= 201703L
```

Comment:
What do you think about providing an alias for that? It's also used a bit further down.

Reply:
Can you elaborate a bit more?

Reply:
Something like `#if CPP17_AVAILABLE` is easier to read for me.